lib/asciidoctor/converter/semantic_html5.rb

```diff
@@ -67,6 +67,150 @@ def convert_paragraph node
     end
   end
 
+  def convert_thematic_break node
+    "<hr />"
+  end
+
+  def convert_admonition node
```

Comment:
I don't want to comment too much on this pull request since we need to extract each `convert_` method in a dedicated pull request.
However, I think we should open an issue where we can discuss how to implement an admonition. In https://github.com/asciidoctor/asciidoctor/issues/242 Dan said:

> Admonition blocks
Admonition blocks are rendered as a table, which makes styling them very difficult. They should be done instead as layered \<div> tags that can accommodate a wider range of layouts.

I don't necessarily agree but I think we should discuss it in an issue before jumping on the implementation.

I'm a bit unsure about `<aside>` as this element is described as: "The [\<aside>](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a portion of a document whose content is only *indirectly related to the document's main content*.".

In my opinion, admonitions are directly related to the content, so we might have to resort on using `<div>`. But let's discuss it in an issue.

Reply:
See #4291 for more on this.